packages/wallet/src/types.ts

```diff
@@ -106,3 +103,8 @@ export interface ObservableWallet {
   submitTx(tx: Cardano.NewTxAlonzo): Promise<void>;
   shutdown(): void;
 }
+
+export type NetworkInfoProviderRequiredByWallet = Pick<
```

Comment:
```suggestion
export type NetworkInfoRequiredByWallet = Pick<
```

Reply:
Improved the names in b151972e847831473ed045af83c70d41a8724164 0e935f274a6153e72cdb62ca25539f5806e58b49 7a85b1a2ecd07945806ff7172164c27cad6a01a2

Much better now, thanks!